test/danetest.c

```diff
@@ -168,7 +166,7 @@ static char *read_to_eol(BIO *f)
     }
 
     /* Trim trailing whitespace */
-    while (n > 0 && isspace(_UC(buf[n - 1])))
+    while (n > 0 && isspace(buf[n - 1]))
```

Comment:
I'm not sure what I was thinking -- this is a transition between being removed by using the new functions and using the old.  I'll revisit.